sdk/identity/azure-identity/tests/test_vscode_credential.py

```diff
@@ -13,7 +13,7 @@
     from unittest import mock
 except ImportError:  # python < 3.3
     import mock
-from azure.identity._credentials.vscode_credential import VSCodeCredential
+from azure.identity._credentials.vscode_credential import VSCodeCredential, get_credentials
```

Comment:
Yes. In my thoughts, we have both linux & windows runs in PR CI. We are good on coverage.

Reply:
I write code on one platform at a time. It would help me to know my harmless refactoring broke something on another platform before I open a pull request.

Reply:
The interface between adapters and vscodecred is the method get_credentials. 

On one side, we mock get_credentials and test the code in vscodecred.

The other side, we test the code in adapters which are platform specific. 

Make sense?

Reply:
What I'm getting at is more about the development inner loop than overall test coverage. This module contains tests that could run on all platforms. If they did, they could tell me I broke something in the Linux or macOS adapter while I'm working in a Windows environment. As is, I won't know until I open a pull request and CI fails.

Reply:
I am not quite clear. Assuming you don't change code in linux & mac adapter (you should not because you are developing on windows. :P). The interface is only get_credentials whose contract is get_credentials() -> str OR none. We have tests for get_credentials returns str & returns none. As long as those tests pass, why could you break linux/mac code?

Reply:
Btw, the code is one way. get_token relies on those adapters but there is no reverse dependency. I cannot imaging how we can break linux/mac code by modifying windows code?

Reply:
The test suite will pass on Windows after I delete `linux_vscode_adapter.py` and `macos_vscode_adapter.py`. I would rather it fail. We can't guarantee the platform-specific code works by running tests on one platform, but it is possible to use `mock` to provide basic coverage on all platforms. That's what I'm suggesting.

Reply:
As I said, you should not modify linux/mac code when you develop on windows. :) 

And I don't think this is a blocker, right? Let's open an issue to track it.

Reply:
#11081 created to track it.